content/en/methods/reports.md

```diff
@@ -24,6 +24,8 @@ aliases: [
 POST /api/v1/reports HTTP/1.1
 ```
 
+Report a problematic account to your moderators.
```

Comment:
This isn't just for reporting problematic accounts, but also for reporting problematic content (statuses), through reporting the account with supporting statuses.